ArcGISRuntimeSDKQt_QMLSamples/EditData/EditFeatureAttachments/README.metadata.json

```diff
@@ -1,45 +1,42 @@
 {
     "category": "Edit data",
-    "description": "The sample demonstrates how to add, delete, and fetch attachments for a specific feature in a feature service",
+    "description": "Add, delete, and download attachments for features from a service.",
     "ignore": false,
     "images": [
         "screenshot.png"
     ],
     "keywords": [
-        "edit feature attachments",
-        "attachments",
-        "edit features",
-        "MapView",
-        "Map",
-        "Basemap",
-        "Viewpoint",
-        "SpatialReference",
-        "ServiceFeatureTable",
+        "Edit and Manage Data",
+        "JPEG",
+        "PDF",
+        "PNG",
+        "TXT",
+        "image",
+        "picture",
+        "ApplyEdits",
+        "AttachmentListModel",
+        "DeleteAttachment",
         "FeatureLayer",
-        "Feature",
-        "ArcGISFeature",
-        "FeatureEditResult",
-        "FeatureQueryResult",
-        "AttachmentListModel"
+        "FetchAttachments",
+        "FetchData",
+        "ServiceFeatureTable",
+        "UpdateFeature"
+    ],
+    "redirect_from": [
+        "/qt/latest/qml/sample-code/sample-qt-editfeatureattachments.htm"
     ],
-    "redirect_from": "/qt/latest/qml/sample-code/sample-qt-editfeatureattachments.htm",
     "relevant_apis": [
-        "MapView",
-        "Map",
-        "Basemap",
-        "Viewpoint",
-        "SpatialReference",
-        "ServiceFeatureTable",
+        "ApplyEdits",
+        "AttachmentListModel",
+        "DeleteAttachment",
         "FeatureLayer",
-        "Feature",
-        "ArcGISFeature",
-        "FeatureEditResult",
-        "FeatureQueryResult",
-        "AttachmentListModel"
+        "FetchAttachments",
+        "FetchData",
+        "ServiceFeatureTable",
+        "UpdateFeature"
     ],
     "snippets": [
-        "EditFeatureAttachments.qml",
-        "main.qml"
```

Comment:
@sbiswas96 I saw from your latest PR on this file that you had to reorder these to fix the bug that would open a separate window at launch. Typically I don't think we include other qml files in here so I removed it and tested it locally and had no problem. Is there a reason why we are including "main.qml" in this case? (came across this due to a merge conflict)

Reply:
@jared-2016 No, it should be fine to remove that file.